### DISCUSSION TITLE
heat transfer through the wall

### DISCUSSION THREAD

**Discussion by ceydabcn** [firemodels/fds#12856] [Q&A]
hello,
I'm trying to ensure heat transfer through the wall, but I couldn't. I defined the reinforced concrete material and assigned it to the surface, but as a result, heat does not pass to the other side of the wall, even though it is exposed. I would be glad if you can help me, my sample file is attached
[example.zip](https://github.com/firemodels/fds/files/15138685/example.zip)
.

**Comment by drjfloyd** [comment]
Please attach your fds input file (.fds) and not your Pyrosim file (.psm). Also what version of FDS are you using?

**Reply by ceydabcn** [reply]
I am an architect and I can only use the Pyrosim program for my thesis, my FDS knowledge is not very sufficient. But I added the fds results.


deneme.fds
Generated by PyroSim 2023.3.1312
Apr 27, 2024, 6:48:08 PM

&HEAD CHID='deneme'/
&TIME T_END=100.0/
&DUMP DT_RESTART=300.0, DT_SL3D=0.25/

&MESH ID='MESH', IJK=17,24,19, XB=80.31635,83.71635,1.601554,6.401554,-0.2,3.6/

&SPEC ID='REAC_FUEL', FORMULA='C1.0H1.7O0.72N0.001'/

&REAC ID='WOOD_OAK',
      FYI='SFPE Handbook, 5th Ed',
      FUEL='REAC_FUEL',
      CO_YIELD=4.0E-3,
      SOOT_YIELD=0.015/

&DEVC ID='Device', QUANTITY='THERMOCOUPLE', XYZ=82.276408,4.232881,1.375/
&DEVC ID='Device04', QUANTITY='THERMOCOUPLE', XYZ=82.364218,4.001554,1.375/

&MATL ID='clt',
      SPECIFIC_HEAT=1.6,
      CONDUCTIVITY=0.13,
      DENSITY=470.0,
      EMISSIVITY=0.8,
      HEAT_OF_COMBUSTION=1.3/
&MATL ID='glass',
      SPECIFIC_HEAT=0.84,
      CONDUCTIVITY=0.8,
      DENSITY=2500.0/
&MATL ID='CONCRETE',
      FYI='NBSIR 88-3752 - ATF NIST Multi-Floor Validation',
      SPECIFIC_HEAT=1.04,
      CONDUCTIVITY=1.8,
      DENSITY=2280.0/
&MATL ID='gazbeton',
      SPECIFIC_HEAT=1.0,
      CONDUCTIVITY=0.09,
      DENSITY=337.0/

&SURF ID='Surface01',
      COLOR='RED',
      HRRPUA=660.0,
      TMP_FRONT=300.0/
&SURF ID='betonarme',
      MLRPUA=0.0,
      IGNITION_TEMPERATURE=100.0,
      BURN_AWAY=.TRUE.,
      MATL_ID(1,1)='CONCRETE',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=0.2/

&OBST ID='Obstruction', XB=80.71635,83.391299,2.101554,6.201554,-0.2,-5.551115E-16/ 
&OBST ID='Obstruction', XB=80.72239,83.391299,2.101554,6.201554,2.75,3.148539/ 
&OBST ID='Obstruction', XB=80.51635,83.71635,4.001554,4.201554,-0.2,3.2, COLOR='WHITE', SURF_ID='betonarme'/ 
&OBST ID='Obstruction', XB=80.71635,80.91635,2.201554,4.001554,-2.664535E-15,2.8/ 
&OBST ID='Obstruction', XB=80.71635,80.91635,4.201554,6.001554,-2.664535E-15,2.8/ 
&OBST ID='Obstruction', XB=80.71635,83.31635,2.001554,2.201554,-2.664535E-15,2.8/ 
&OBST ID='Obstruction', XB=80.71635,83.31635,6.001554,6.201554,-2.664535E-15,2.8/ 
&OBST ID='Obstruction', XB=81.31635,81.51635,5.801554,6.001554,-0.2,0.6, SURF_IDS='Surface01','INERT','INERT'/ 
&OBST ID='Obstruction', XB=81.31635,81.91635,5.601554,5.801554,-0.2,0.6, SURF_IDS='Surface01','INERT','INERT'/ 
&OBST ID='Obstruction', XB=81.31635,82.11635,5.201554,5.601554,-0.2,0.6, SURF_IDS='Surface01','INERT','INERT'/ 
&OBST ID='Obstruction', XB=81.71635,82.11635,5.001554,5.201554,-0.2,0.6, SURF_IDS='Surface01','INERT','INERT'/ 
&OBST ID='Obstruction', XB=83.11635,83.31635,2.201554,4.001554,-2.664535E-15,2.8/ 
&OBST ID='Obstruction', XB=83.11635,83.31635,4.201554,6.001554,-2.664535E-15,2.8/ 
&OBST ID='Obstruction', XB=81.11635,81.31635,5.401554,5.401554,-0.2,0.6/ 
&OBST ID='Obstruction', XB=82.11635,82.31635,5.601554,5.601554,-0.2,0.6/ 

&HOLE ID='Hole', XB=81.157449,82.027832,6.001554,6.201554,1.721845,2.160129/ 
&HOLE ID='Hole', XB=83.191299,83.391299,2.937765,3.668095,0.937949,2.033345/ 

&VENT ID='Mesh Vent: MESH [XMAX]', SURF_ID='OPEN', XB=83.71635,83.71635,1.601554,6.401554,-0.2,3.6/ 
&VENT ID='Mesh Vent: MESH [XMIN]', SURF_ID='OPEN', XB=80.31635,80.31635,1.601554,6.401554,-0.2,3.6/ 
&VENT ID='Mesh Vent: MESH [YMAX]', SURF_ID='OPEN', XB=80.31635,83.71635,6.401554,6.401554,-0.2,3.6/ 
&VENT ID='Mesh Vent: MESH [YMIN]', SURF_ID='OPEN', XB=80.31635,83.71635,1.601554,1.601554,-0.2,3.6/ 
&VENT ID='Mesh Vent: MESH [ZMAX]', SURF_ID='OPEN', XB=80.31635,83.71635,1.601554,6.401554,3.6,3.6/ 
&VENT ID='Mesh Vent: MESH [ZMIN]', SURF_ID='OPEN', XB=80.31635,83.71635,1.601554,6.401554,-0.2,-0.2/ 

&SLCF QUANTITY='TEMPERATURE', ID='Slice01', PBX=47.791155/
&SLCF QUANTITY='TEMPERATURE', ID='Slice02', PBZ=4.541939/
&SLCF QUANTITY='TEMPERATURE', ID='Slice03', PBY=16.461539/
&SLCF QUANTITY='TEMPERATURE', ID='Slice04', PBX=85.730331/
&SLCF QUANTITY='TEMPERATURE', ID='Slice', PBX=81.756186/
&SLCF QUANTITY='TEMPERATURE', ID='Temperature', XB=30.986124,52.186124,1.779612,15.979612,-1.475022,10.324978/
&SLCF QUANTITY='TEMPERATURE', ID='3D Slice', XB=80.31635,83.71635,1.601554,6.401554,-0.2,2.6/


&TAIL /

**Reply by ceydabcn** [reply]
I just want the temperature to be transmitted to the other side of the wall to test the material difference, to understand how different transition is. However, unlike insulated, it does not provide heat transfer to the other room.

I would be very grateful if you could help me, thank you.

**Reply by drjfloyd** [reply]
You are running for 100 s and you have a 20 cm thick layer of concrete.  The thermal penetration time of 20 cm of concrete is much larger than 100 s.

**Reply by ceydabcn** [reply]
I run the simulation for 500 seconds and result was the same. You can see the heatmap of the simulation in the screenshot below. Even though fire ends the heat doesnt transter to the other room. Can you help me find the issue?


![01](https://github.com/firemodels/fds/assets/168291614/cf42adf4-f9bb-49ba-8a49-5606feb7212f)


Betonarme_-denemeCopy_-_Copy.fds
Generated by PyroSim 2023.3.1312
Apr 28, 2024, 8:13:15 PM

&HEAD CHID='Betonarme_-denemeCopy_-_Copy'/
&TIME T_END=500.0/
&DUMP DT_RESTART=300.0, DT_SL3D=0.25/

&MESH ID='MESH', IJK=35,47,30, XB=52.29314,59.29314,7.69365,17.09365,-0.8,5.2/

&SPEC ID='REAC_FUEL', FORMULA='C1.0H1.7O0.72N0.001'/

&REAC ID='WOOD_OAK',
      FYI='SFPE Handbook, 5th Ed',
      FUEL='REAC_FUEL',
      CO_YIELD=4.0E-3,
      SOOT_YIELD=0.015/

&DEVC ID='Device', QUANTITY='THERMOCOUPLE', XYZ=56.230107,12.462549,1.375/
&DEVC ID='Device04', QUANTITY='THERMOCOUPLE', XYZ=56.664668,12.262549,1.324235/

&MATL ID='clt',
      SPECIFIC_HEAT=1.6,
      CONDUCTIVITY=0.13,
      DENSITY=470.0,
      EMISSIVITY=0.8,
      HEAT_OF_COMBUSTION=1.3/
&MATL ID='glass',
      SPECIFIC_HEAT=0.84,
      CONDUCTIVITY=0.8,
      DENSITY=2500.0/
&MATL ID='CONCRETE',
      FYI='NBSIR 88-3752 - ATF NIST Multi-Floor Validation',
      SPECIFIC_HEAT=1.04,
      CONDUCTIVITY=1.8,
      DENSITY=2280.0/
&MATL ID='gazbeton',
      SPECIFIC_HEAT=1.0,
      CONDUCTIVITY=0.09,
      DENSITY=337.0/

&SURF ID='Surface01',
      COLOR='RED',
      HRRPUA=660.0,
      TMP_FRONT=300.0/
&SURF ID='betonarme',
      HEAT_TRANSFER_COEFFICIENT=1.75,
      MATL_ID(1,1)='CONCRETE',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=0.2/

&OBST ID='Obstruction', XB=53.939098,58.44251,12.262549,12.462549,0.0,2.75, SURF_ID='betonarme'/ 
&OBST ID='Obstruction', XB=54.29314,54.49314,8.49365,16.09365,-0.2,3.0/ 
&OBST ID='Obstruction', XB=54.29314,58.09314,8.29365,8.49365,-0.2,3.0/ 
&OBST ID='Obstruction', XB=54.29314,58.09314,16.09365,16.29365,-0.2,3.0/ 
&OBST ID='Obstruction', XB=54.49314,55.89314,14.09365,14.69365,-0.2,-2.775558E-16/ 
&OBST ID='Obstruction', XB=54.49314,57.89314,8.49365,14.09365,-0.2,-2.775558E-16/ 
&OBST ID='Obstruction', XB=54.49314,57.89314,14.69365,16.09365,-0.2,-2.775558E-16/ 
&OBST ID='Obstruction', XB=54.49314,57.89314,8.49365,16.09365,2.8,3.0/ 
&OBST ID='Obstruction', XB=55.89314,56.69314,14.09365,14.69365,-0.2,0.6, SURF_IDS='Surface01','INERT','INERT'/ 
&OBST ID='Obstruction', XB=56.69314,57.89314,14.09365,14.69365,-0.2,-2.775558E-16/ 
&OBST ID='Obstruction', XB=57.89314,58.09314,8.49365,16.09365,-0.2,3.0/ 

&VENT ID='Mesh Vent: MESH [XMAX]', SURF_ID='OPEN', XB=59.29314,59.29314,7.69365,17.09365,-0.8,5.2/ 
&VENT ID='Mesh Vent: MESH [XMIN]', SURF_ID='OPEN', XB=52.29314,52.29314,7.69365,17.09365,-0.8,5.2/ 
&VENT ID='Mesh Vent: MESH [YMAX]', SURF_ID='OPEN', XB=52.29314,59.29314,17.09365,17.09365,-0.8,5.2/ 
&VENT ID='Mesh Vent: MESH [YMIN]', SURF_ID='OPEN', XB=52.29314,59.29314,7.69365,7.69365,-0.8,5.2/ 
&VENT ID='Mesh Vent: MESH [ZMAX]', SURF_ID='OPEN', XB=52.29314,59.29314,7.69365,17.09365,5.2,5.2/ 
&VENT ID='Mesh Vent: MESH [ZMIN]', SURF_ID='OPEN', XB=52.29314,59.29314,7.69365,17.09365,-0.8,-0.8/ 

&SLCF QUANTITY='TEMPERATURE', ID='Slice01', PBX=47.791155/
&SLCF QUANTITY='TEMPERATURE', ID='Slice02', PBZ=1.254929/
&SLCF QUANTITY='TEMPERATURE', ID='Slice03', PBY=16.461539/
&SLCF QUANTITY='TEMPERATURE', ID='Slice04', PBX=85.730331/
&SLCF QUANTITY='TEMPERATURE', ID='Slice', PBX=56.28742/
&SLCF QUANTITY='TEMPERATURE', ID='3D Slice', XB=52.39314,59.29314,7.69365,17.09365,-0.6,5.1/


&TAIL /

**Reply by drjfloyd** [reply]
500 s is still a very short period of time for 20 cm of concrete. Calculate the thermal penetration depth for your wall at various times to see how long it will take for heat to transfer through the wall.

For testing purposes, to see if your basic approach is working, change to something that will quickly trasnfer heat. Like a thin piece of metal.
